cli/mender-artifact/copy.go

```diff
@@ -85,10 +93,7 @@ func Copy(c *cli.Context) (err error) {
 		return cli.NewExitError(err.Error(), 1)
 	}
 
-	if err = w.Close(); err != nil {
-		return err
-	}
-	return r.Close()
+	return nil
```

Comment:
If we will assign the 'err' named return variable correctly we should skip this one.